vm/procedures/src/jsonrpc/loadStateProcedure.js

```diff
@@ -0,0 +1,24 @@
+import { loadStateHandler } from '../index.js'
+
+/**
+ * Creates a DumpState JSON-RPC Procedure for handling dumpState requests with Ethereumjs EVM
+ * @param {import('@tevm/state').TevmStateManager} stateManager
+ * @returns {import('@tevm/api').LoadStateJsonRpcProcedure}
+ */
+export const loadStateProcedure = (stateManager) => async (request) => {
+	const {
+		params: { state },
+	} = request
+
+	await loadStateHandler(stateManager)({ state: state })
```

Comment:
```suggestion
	await loadStateHandler(stateManager)({ state: state })
```
Since this json rpc request will have hex strings instead of bigints this will need to be decoded back into bigints

Reply:
Should it be decoded in the procedure or handler?

Reply:
Nvm I see in our discord chat 

Implement procedure in procedure/src/procedures/*
  This procedure should deserialize the request and pass it off to the handler. In your case this means turning a bunch of hex strings into bigints

Reply:
Does this still have any bigints in it?

Reply:
![Screenshot 2024-01-16 at 11 23 29 PM](https://github.com/evmts/tevm-monorepo/assets/95137353/508525ed-9ce9-4a5b-94aa-cd2aad5845a0)
Super weird V8 had these changes then at some point there were written over/dropped.